src/paper/openalex_util.py

```diff
@@ -156,18 +158,42 @@ def process_openalex_works(works):
             sentry.log_error(e, message="Failed to bulk update papers")
 
     # Upsert concepts and associate to papers
-    for paper_id, topics_and_concepts in paper_to_topics_and_concepts.items():
-        openalex_concepts, openalex_topics = topics_and_concepts
-        create_paper_related_tags(paper_id, openalex_concepts, openalex_topics)
+    for paper_id, paper_data in paper_to_openalex_data.items():
+        print("processing work: " + paper_data["openalex_work"]["id"])
```

Comment:
I am wondering if we want these type of side-effects in a utility function, and if so, whether we should use an actual logger that includes some contextual metadata (log level, timestamp, formatting at a minimum) . It's fully understood that `print`s have spread across the code base, but I suggest we stop this practice.

Reply:
Let's talk about a better way to do this during our sync